src/GraphQL/MissingRequestServicesException.cs

```diff
@@ -1,9 +1,13 @@
 using System;
 
-namespace GraphQL.MicrosoftDI
+namespace GraphQL
 {
-    internal class MissingRequestServicesException : InvalidOperationException
+    /// <summary>
+    /// Indicates that <see cref="IResolveFieldContext.RequestServices"/> was required but not set.
+    /// </summary>
+    public class MissingRequestServicesException : InvalidOperationException
```

Comment:
Needed by both MicrosoftDI and main library, so I moved it from MicrosoftDI and made it public.

Reply:
ok